### DISCUSSION TITLE
Shorthand properties exhaustive list

### DISCUSSION THREAD

**Discussion by RemyMachado** [chakra-ui/panda#1722] [General]
I was wondering what are and aren't usable shorthand properties in Pandacss. I didn't find an exhaustive list in the [documentation](https://panda-css.com/docs/concepts/writing-styles#shorthand-properties). I have no experience with tailwind or any similar libraries and I feel like I have to guess what exists and what doesn't.

> Properties like `borderRadius`, `backgroundColor`, and `padding` can be swapped to their shorthand equivalent `rounded`, `bg`, and `p`.

### What do you think about explicitly explaining what shorthand properties are available?

If it's the same as an other library, maybe create a link to their documentation.

**Comment by anubra266** [comment]
@RemyMachado We document them in the utilities section with props
https://panda-css.com/docs/utilities/background

**Reply by RemyMachado** [reply]
Great, thanks! :raised_hands: 

Maybe, making a reference to them in the `writing styles` section would help.
